<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorCalltipsPage</class>
 <widget class="QWidget" name="EditorCalltipsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>373</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Calltips&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line18">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ctEnabledCheckBox">
     <property name="toolTip">
      <string>Select this to enable calltips</string>
     </property>
     <property name="text">
      <string>Automatic Calltips Enabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Visible Calltips</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="ctVisibleSlider">
        <property name="toolTip">
         <string>Move to set the maximum number of calltips shown (0 = all available)</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lCDNumber5">
        <property name="toolTip">
         <string>Displays the maximum number of calltips to be shown</string>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calltipsPositionBox">
     <property name="title">
      <string>Calltips Position</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="positionComboBox">
        <property name="toolTip">
         <string>Select the position for the calltips</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Colours</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="TextLabel2_2_2_2">
        <property name="text">
         <string>Background colour:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calltipsBackgroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for calltips.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plug-In Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ctScintillaCheckBox">
        <property name="toolTip">
         <string>Select to show QScintilla provided calltips, if the selected plug-ins fail</string>
        </property>
        <property name="whatsThis">
         <string>Qscintilla provided calltips are shown, if this option is enabled and calltips shall be provided by plug-ins (see calltips sub-page of the plug-in) and the plugin-ins don't deliver any calltips.</string>
        </property>
        <property name="text">
         <string>Show QScintilla calltips, if plug-ins fail</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>388</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ctEnabledCheckBox</tabstop>
  <tabstop>calltipsBackgroundButton</tabstop>
  <tabstop>ctScintillaCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ctVisibleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lCDNumber5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
